### DISCUSSION TITLE
**Wordpress** Webcams don't show for subscribers on moderators screen

### DISCUSSION THREAD

**Discussion by KW-IT** [mynaparrot/plugNmeet-server#228] [General]
Hello, 

My apologies if this is the wrong spot to ask

I have an issue where subscriber webcams will not show on moderators' screens, but when other moderators are on they will show up on each other's screens and can see each other's webcams. 

Is this not working due to the room settings or is it the Plug-in itself

**Comment by jibon57** [comment]
Which browser are you trying from? Did you open screen share first then webcam?

**Reply by KW-IT** [reply]
Whoops, I forgot to check a different browser initially lol, the issue for us seems to be firefox mobile because Chrome mobile seems to work just fine. 

The camera was turned on directly after the subscriber was let into the room 

Both firefox and chrome mobile/desktops have the latest updates.

**Reply by jibon57** [reply]
Glad to hear that you figure it out. Webrtc in mobile yet not very stable that's why need to try from different browsers. Google Chrome most of the time works better. For iOS safari.

**Reply by KW-IT** [reply]
ahh ok, that is good to know.

Thank you for the help. PlugnMeet is awesome
